### ISSUE TITLE
Inconsistent enum character casing between enum file and other generated files

### ISSUE THREAD

**Issue by point-source** [epam-cross-platform-lab/swagger-dart-code-generator#198]
I am generating code based on the Plaid openapi spec here: https://github.com/plaid/plaid-openapi/blob/master/2020-09-14.yml

Most of it looks pretty good but one of the enums, "AprAprType" appears as "enums.APRAprType" in the plaid.swagger.dart file. This does not match the plaid.enums.swagger.dart file due to the inconsistent character casing on the first Apr/APR.

This has been generated using the [2.0.9 branch directly from the repo on this commit](https://github.com/epam-cross-platform-lab/swagger-dart-code-generator/tree/f15eebaf5021035a7b3871a14fa60129c58ac2de).

Enum declaration:
```dart
enum AprAprType {
  @JsonValue('swaggerGeneratedUnknown')
  swaggerGeneratedUnknown,
  @JsonValue('balance_transfer_apr')
  balanceTransferApr,
  @JsonValue('cash_apr')
  cashApr,
  @JsonValue('purchase_apr')
  purchaseApr,
  @JsonValue('special')
  special
}
```

Attempted enum use:
```dart
String? aPRAprTypeToJson(enums.APRAprType? aPRAprType) {
  return enums.$APRAprTypeMap[aPRAprType];
}

enums.APRAprType aPRAprTypeFromJson(String? aPRAprType) {
  if (aPRAprType == null) {
    return enums.APRAprType.swaggerGeneratedUnknown;
  }

  return enums.$APRAprTypeMap.entries
      .firstWhere(
          (element) => element.value.toLowerCase() == aPRAprType.toLowerCase(),
          orElse: () =>
              const MapEntry(enums.APRAprType.swaggerGeneratedUnknown, ''))
      .key;
}

List<String> aPRAprTypeListToJson(List<enums.APRAprType>? aPRAprType) {
  if (aPRAprType == null) {
    return [];
  }

  return aPRAprType.map((e) => enums.$APRAprTypeMap[e]!).toList();
}

List<enums.APRAprType> aPRAprTypeListFromJson(List? aPRAprType) {
  if (aPRAprType == null) {
    return [];
  }

  return aPRAprType.map((e) => aPRAprTypeFromJson(e.toString())).toList();
}
```

**Comment by Vovanella95** [comment]
Hi @point-source ,

I'll check all your issues this week and try to fix it.

**Comment by point-source** [comment]
@Vovanella95 thanks! Sorry about the barrage of reports. Not trying to overwhelm you. Just working on multiple API-related projects at once and this is my favorite generator so far. Thanks for the hard work!

**Comment by Vovanella95** [comment]
@point-source 
Will check all your issues in 3-4 days

**Comment by point-source** [comment]
This appears to have been fixed in release 2.0.8+1. Closing
